apps/mymedicare_cb/tests/test_callback.py

```diff
@@ -155,9 +155,7 @@ def sls_user_info_mock(url, request):
             return {
                 'status_code': 200,
                 'content': {
-                    'sub': '0123456789abcdefghijklmnopqrstuvwxyz',
-                    'given_name': '',
-                    'family_name': '',
```

Comment:
Removing these because SLS does not guarantee the fields are present based on their test cases.